applications/Cere_Turnkey_Private_Blockchain_Network.md

```diff
@@ -47,60 +49,52 @@ There will be three primary directories in this repository:
 ## Team :busts_in_silhouette:
 
 * **Team Members:** Evgeny Svirsky, Shivam C, Huy Thanh N
-* **Team Website:** https://cerebellum-network.github.io/#/team
-* **Code Repos:** https://github.com/Cerebellum-Network/enterprise-blockchain-solution
+* **Team Website:** https://cere.network/#/team
+* **Code Repos:** https://github.com/Cerebellum-Network/private-standalone-network-node.git
 * **Website:**	http://cere.network/
 * **Legal Structure:** Delaware C Corp
 * **Team's Experience:**   
 Evgeny has been one of the core developers on Cere team, where he’s been instrumental in developing our blockchain network, tooling, and integrating with real-world applications.   
 Shivam started programming at the age of 13, with roots in all-round programming. One of the first employees at Asia’s top cryptocurrency exchange, winner of multiple global hackathons and an early Blockchain adopter, believer and developer.   
 Thanh has years of experience working with various blockchain projects ranging from Solidity smart contracts to enterprise integrations with Hyperledger and other networks.   
 
-## Development Roadmap :nut_and_bolt: 
+## Development Roadmap :nut_and_bolt:
 
 * **Total Estimated Duration:** 9 weeks
 * **Total Costs:** 1.8 BTC
 
 ### Milestone 1
 
-* **Estimated Duration:** 5 weeks 
-* **Costs:** 1 BTC
-* **Main Goal:** Basic functionality: Private node, Smart Contract implementation and setup guide.
-
```

Comment:
Keep these please, but adjust the cost of M1 & M2

Reply:
Thanks for your comment, @mmagician .
Updated.